examples/flatland/feedforward/decentralised/run_madqn.py

```diff
@@ -86,37 +87,48 @@ def main(_: Any) -> None:
         time_delta=log_every,
     )
 
+    num_executors = 8
+
+    # exploration_scheduler_fn={
+    #     "executor_0": {
+    #         "train_0": LinearExplorationScheduler(
+    #             epsilon_start=1.0, epsilon_min=0.05, epsilon_decay=1e-4
+    #         ),
+    #         "train_1": LinearExplorationScheduler(
+    #             epsilon_start=1.0, epsilon_min=0.06, epsilon_decay=2e-4
+    #         ),
+    #         "train_2": LinearExplorationScheduler(
+    #             epsilon_start=1.0, epsilon_min=0.07, epsilon_decay=3e-4
+    #         ),
+    #     },
+    #     "executor_1": {
+    #         "train_0": LinearExplorationScheduler(
+    #             epsilon_start=1.0, epsilon_min=0.08, epsilon_decay=4e-4
+    #         ),
+    #         "train_1": LinearExplorationScheduler(
+    #             epsilon_start=1.0, epsilon_min=0.09, epsilon_decay=5e-4
+    #         ),
+    #         "train_2": LinearExplorationScheduler(
+    #             epsilon_start=1.0, epsilon_min=0.10, epsilon_decay=6e-4
+    #         ),
+    #     },
+    # },
```

Comment:
Try to avoid commenting code, if it is not needed anymore you can just delete it. If is needed again some time, you  can still find it in git history